packages/kbn-palettes/classes/gradient_palette.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import chroma, { Scale } from 'chroma-js';
+import { Optional } from 'utility-types';
+import { IKbnPalette } from './types';
+import { KbnBasePalette, KbnBasePaletteConfig } from './palette';
+
+export interface KbnGradientPaletteConfig extends Optional<KbnBasePaletteConfig, 'colorCount'> {
+  colors: string[];
+}
+
+export class KbnGradientPalette extends KbnBasePalette implements IKbnPalette {
```

Comment:
This actually is not used can we delete this?

Reply:
Yeah this was for the custom gradients before I switched to eui functions. Removed in 016f1615074a66354d91174004f5b9ea31ef4a56.